eng/Versions.props

```diff
@@ -20,7 +20,7 @@
     <ReportGeneratorVersion>4.3.6</ReportGeneratorVersion>
 
     <!-- Roslyn -->
-    <MicrosoftCodeAnalysisVersion>3.0.0</MicrosoftCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisVersion>3.5.0</MicrosoftCodeAnalysisVersion>
```

Comment:
This needs to be reverted - this is bumping up the version used to build all analyzer packages in this repo, which is not required given no new API is being consumed. Instead please rename `MicrosoftCodeAnalysisForShippedApisVersion` to `MicrosoftCodeAnalysisForTestsVersion`, update that one to `3.5.0` and then update the relevant test projects to explicit reference `MicrosoftCodeAnalysisForTestsVersion`